terraform/deploy-terraform.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+#This script install and runs terraform and supporting tools to deploy a complete
+#serverless web project to aws using cloudfront/s3/lambda
+
+# Usage deploy-terraform.sh <AWS_ACCESS_KEY_ID> <AWS_SECRET_ACCESS_KEY> <releaseNumber> <releaseEnvironment> <local or octo>
+
+if [ "$5" = "octo" ]
+  then
+    runningInOctoDeploy=true
+  else
+    runningInOctoDeploy=false
+fi
+
+echo "setting release to $3 and deploying to environment $4"
+releaseNumber=$3
```

Comment:
Can you declare all the 5 variables at the top?

Reply:
yep - done